tests/test_analysis/test_feature_relevance/test_relevance.py

```diff
@@ -8,11 +8,21 @@ def test_statistics_relevance_table(simple_df_relevance):
     rt = StatisticsRelevanceTable()
     assert not rt.greater_is_better
     df, df_exog = simple_df_relevance
-    assert rt(df=df, df_exog=df_exog).shape == (2, 2)
+    assert rt(df=df, df_exog=df_exog, return_ranks=False).shape == (2, 2)
 
 
 def test_model_relevance_table(simple_df_relevance):
     rt = ModelRelevanceTable()
     assert rt.greater_is_better
     df, df_exog = simple_df_relevance
-    assert rt(df=df, df_exog=df_exog, model=DecisionTreeRegressor()).shape == (2, 2)
+    assert rt(df=df, df_exog=df_exog, return_ranks=False, model=DecisionTreeRegressor()).shape == (2, 2)
+
+
+def test_relevance_table_ranks(simple_df_relevance):
```

Comment:
Better to make a fixture for the table with ranks and test the method in two cases of `greater_is_better`